install/install.py

```diff
@@ -126,17 +126,21 @@ def setup_virtualenv():
 
     # Update the os environment variable
     os.environ.update(env)
-    if os.path.join(os.environ["WORKON_HOME"], "owtf") == os.environ["VIRTUAL_ENV"]:
-        # Add source to shell config file only if not present
-        Colorizer.info("[*] Adding virtualenvwrapper source to shell config file")
-        shell_rc_path = os.path.join(os.environ["HOME"], ".%src" % os.environ["SHELL"].split(os.sep)[-1])
-        with open(shell_rc_path, "r") as f:
-            if mmap.mmap(f.fileno(), 0, access=mmap.ACCESS_READ).find(source) == -1:
-                run_command("echo '%s' >> %s" % (source, shell_rc_path))
-            else:
-                Colorizer.info("[+] Source line already added to the $SHELL config ")
-    else:
-        Colorizer.warning("Unable to setup virtualenv...")
+    try:
+        if os.path.join(os.environ["WORKON_HOME"], "owtf") == os.environ["VIRTUAL_ENV"]:
+            # Add source to shell config file only if not present
+            Colorizer.info("[*] Adding virtualenvwrapper source to shell config file")
+            shell_rc_path = os.path.join(os.environ["HOME"], ".%src" % os.environ["SHELL"].split(os.sep)[-1])
+            with open(shell_rc_path, "r") as f:
+                if mmap.mmap(f.fileno(), 0, access=mmap.ACCESS_READ).find(source) == -1:
+                    run_command("echo '%s' >> %s" % (source, shell_rc_path))
+                else:
+                    Colorizer.info("[+] Source line already added to the $SHELL config ")
+        else:
+            return False
+    except:
```

Comment:
Please don't do that. Either you catch for a specific exception or you let it raise. Avoid catching all exceptions "just to make sure".